website/docs/docs/partnerships.mdx

```diff
@@ -9,38 +9,30 @@ description: >-
 
 The Terraform Integration Program facilitates prospect partners in creating and publishing Terraform integrations that have been verified by HashiCorp.
 
-## Terraform Editions
-
-Terraform is an infrastructure as code (IaC) tool that allows you to build, change, and version infrastructure safely and efficiently. This includes low-level components such as compute instances, storage, and networking, as well as high-level components such as DNS entries, SaaS features, etc. Terraform can manage both existing service providers and custom in-house solutions.
-
-HashiCorp offers three editions of Terraform: Open Source, Terraform Cloud, and Terraform Enterprise.
-
-- [Terraform Open Source](/) provides a consistent CLI workflow to manage hundreds of cloud services. Terraform codifies cloud APIs into declarative configuration files.
-- [Terraform Cloud (TFC)](/cloud) is a free to use, self-service SaaS platform that extends the capabilities of the open source Terraform CLI. It adds automation and collaboration features, and performs Terraform functionality remotely, making it ideal for collaborative and production environments. Terraform Cloud is available as a hosted service at https://app.terraform.io. Small teams can sign up for free to connect Terraform to version control, share variables, run Terraform in a stable remote environment, and securely store remote state. Paid tiers allow you to add more than five users, create teams with different levels of permissions, enforce policies before creating infrastructure, and collaborate more effectively.
-- [Terraform Enterprise (TFE)](/enterprise) is our self-hosted distribution of Terraform Cloud with advanced security and compliance features. It offers enterprises a private instance that includes the advanced features available in Terraform Cloud.
-
 ## Types of Terraform Integrations
 
 The Terraform ecosystem is designed to enable users to apply Terraform across different use cases and environments. The Terraform Integration Program currently supports both workflow and integration partners (details below). Some partners can be both, depending on their use cases.
 
 - **Workflow Partners** build integrations for Terraform Cloud and/or Terraform Enterprise. Typically, these partners are seeking to enable customers to use their existing platform within a Terraform Run.
```

Comment:
```suggestion
- **Workflow Partners** build integrations for Terraform Cloud, Terraform Enterprise, or both. Typically, these partners seek to enable customers to use their existing platform within a Terraform Run.
```

Suggest avoiding "and/or" and suggest changing "are seeking" to "seek" to use fewer words.